src/dlangui/graphics/colors.d

```diff
@@ -191,20 +190,26 @@ uint makeRGBA(T)(T r, T g, T b, T a) pure nothrow {
 
 /// blend two RGB pixels using alpha
 uint blendARGB(uint dst, uint src, uint alpha) pure nothrow {
-    uint dstalpha = dst >> 24;
-    if (dstalpha > 0x80)
-        return src;
+    uint srca = (src >> 24) & 0xFF;
+    uint dsta = (dst >> 24) & 0xFF;
+
     uint srcr = (src >> 16) & 0xFF;
     uint srcg = (src >> 8) & 0xFF;
     uint srcb = (src >> 0) & 0xFF;
+
     uint dstr = (dst >> 16) & 0xFF;
     uint dstg = (dst >> 8) & 0xFF;
     uint dstb = (dst >> 0) & 0xFF;
+
     uint ialpha = 255 - alpha;
+
     uint r = ((srcr * ialpha + dstr * alpha) >> 8) & 0xFF;
     uint g = ((srcg * ialpha + dstg * alpha) >> 8) & 0xFF;
     uint b = ((srcb * ialpha + dstb * alpha) >> 8) & 0xFF;
-    return (r << 16) | (g << 8) | b;
+
+    uint a = ((srca * ialpha + dsta * alpha) >> 8) & 0xFF;
+
+    return (a << 24) | (r << 16) | (g << 8) | b;
```

Comment:
I see that the formula changed. I'm not good with this kind of thing, so can you please elaborate on what exactly was wrong, and how this fixes the problem?
It would be great if you could write additional unittests.

Reply:
I added blending of alpha channels. It was missing. For RGBA colors, blending worked as it did for RGB because of this (without alpha channel). Now it works correctly both with and without transparency. I'll try to add tests this week when I have time.

Reply:
This code calculates the mixed alpha, then adds it to the final color (ARGB).

Reply:
Can you please show a screenshot of the difference so I understand better what's going on?

And, I would still like some unittests to make this behavior fixed. [Not in a sense of it being broken, but in a sense of it being fundamental]